### PR TITLE
[ROCKETMQ-90] Include client IP in consumerProgress command output

### PR DESCRIPTION
**Pull request by lizhanhui**
Include client IP per message queue of consumer progress command output

### CODE REVIEW

**Comment by lizhanhui** [comment]
@shroman @vongosling  Any idea on this PR?

File: tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerProgressSubCommand.java [hunk 1]
```diff
@@ -94,12 +117,14 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook) {
...
                     }
-                    System.out.printf("%-32s  %-32s  %-4d  %-20d  %-20d  %-20d  %s%n",
+
+                    System.out.printf("%-32s  %-32s  %-4d  %-20d  %-20d  %-20s %-20d  %s%n",
                         UtilAll.frontStringAtLeast(mq.getTopic(), 32),
                         UtilAll.frontStringAtLeast(mq.getBrokerName(), 32),
                         mq.getQueueId(),
                         offsetWrapper.getBrokerOffset(),
                         offsetWrapper.getConsumerOffset(),
+                        messageQueueAllocationResult.containsKey(mq) ? messageQueueAllocationResult.get(mq) : "NA",
```
Comment (shroman) [inline tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerProgressSubCommand.java]: Not that important here, but you can do the same with accessing `messageQueueAllocationResult` only once. `get()` and see if it is null.

**Comment by vongosling** [comment]
LGTM~

**Comment by lizhanhui** [comment]
@shroman Please re-check and  help merge this issue if it's all right.

**Comment by shroman** [comment]
@lizhanhui +1

**Comment by zhouxinyu** [comment]
@lizhanhui why did we close this PR without merge?

**Comment by lizhanhui** [comment]
@zhouxinyu it's been merged.

**Pull request closed without merging**